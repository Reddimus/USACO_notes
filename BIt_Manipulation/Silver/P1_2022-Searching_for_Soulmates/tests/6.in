10
2387721793 2140797847
986376326 2896264937
257836810 2718208863
3880528011 89077818
1795184852 76469283
715223075 2192106427
887062135 3359636024
3591388742 3432279954
3051649863 2598997240
1807296971 963105519
