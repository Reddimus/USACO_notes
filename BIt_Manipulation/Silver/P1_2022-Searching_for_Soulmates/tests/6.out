95
88
80
84
80
90
84
93
88
80
